### ISSUE TITLE
npm package for 3.4.0 is missing files

### ISSUE THREAD

**Issue by ashvinsrivatsa** [prettier/plugin-xml#777]
Version 3.4.0 of this package [on npm](https://www.npmjs.com/package/@prettier/plugin-xml/v/3.4.0?activeTab=code) appears to be erroneously missing a bunch of files that were present in the prior version (3.3.1). 

The `src/` directory only contains `plugin.js`, which tries to import non-existent files like `./language.js` and consequently crashes.

**Comment by regseb** [comment]
When I do [`npm pack`](https://docs.npmjs.com/cli/v10/commands/npm-pack) in plugin-xml  project: all `src` files are in tarball. Normally, files should also be published in npm. Perhaps it need to use the [`"files"`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files) property to _force_ their inclusion.

```
> @prettier/plugin-xml@3.4.0 prepare
> node bin/languages.js && husky install

(node:12920) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
install command is deprecated
npm notice 
npm notice 📦  @prettier/plugin-xml@3.4.0
npm notice === Tarball Contents === 
npm notice 8.2kB  CHANGELOG.md      
npm notice 3.4kB  CODE_OF_CONDUCT.md
npm notice 1.1kB  LICENSE           
npm notice 5.8kB  README.md         
npm notice 1.8kB  package.json      
npm notice 4.1kB  src/embed.js      
npm notice 5.8kB  src/languages.js  
npm notice 5.9kB  src/parser.js     
npm notice 2.1kB  src/plugin.js     
npm notice 15.8kB src/printer.js    
npm notice 94B    types/plugin.d.ts 
npm notice === Tarball Details === 
npm notice name:          @prettier/plugin-xml                    
npm notice version:       3.4.0                                   
npm notice filename:      prettier-plugin-xml-3.4.0.tgz           
npm notice package size:  15.0 kB                                 
npm notice unpacked size: 54.0 kB                                 
npm notice shasum:        1ee3008cb59d202d64aad499a37e704e336c0fd9
npm notice integrity:     sha512-dIBTaLmuvnEKb[...]vgcMmV6bX+bGQ==
npm notice total files:   11                                      
npm notice 
prettier-plugin-xml-3.4.0.tgz
```

**Comment by kddnewton** [comment]
Sorry about that, I just published v3.4.1 which should have this all fixed up.
